### DISCUSSION TITLE
Can I move a nuget package to a new repository?

### DISCUSSION THREAD

**Discussion by derekssmith** [community/community#26090] [Actions]
<p>I recently archived a dotnet project that was posting a Nuget package to the Github package registry.  I moved the library to a new Github repo in the same organization.  How can I move this package from the archived repository to the new repository, so I can continue to publish it from the new location?</p>

**Comment by derekssmith** [comment]
<p>Any thoughts on this?</p>

**Comment by alexandre-machado** [comment]
<p>does anyone else need this?</p>

**Comment by jcansdale** [comment, marked as answer]
<p>Hi <a class="mention" href="/derekssmith">@derekssmith</a>,</p>
<aside class="quote no-group" data-username="derekssmith" data-post="1" data-topic="14832">
<div class="title">
<div class="quote-controls"></div>
 derekssmith:</div>
<blockquote>
<p>How can I move this package from the archived repository to the new repository, so I can continue to publish it from the new location?</p>
</blockquote>
</aside>
<p>Can you try the following:</p>
<ol>
<li>Download all versions of your .nupkg</li>
<li>Make the original repository private</li>
<li>Delete all package versions</li>
<li><code>dotnet tool install update gpr -g</code></li>
<li>
<code>gpr push *.nupkg --repository &lt;OWNER/NEW_REPO&gt;</code> -k &lt;PAT with repo, read:packages and write:packages scopes&gt;</li>
</ol>
<p>You can now publish new packages to the new <code>OWNER/NEW_REPO</code> repository. If you don’t need to keep your old packages, you can skip steps 1, 4 and 5.</p>
<p>I hope that helps!</p>

**Comment by Leaxe** [comment]
Not sure when this was added, but you can change the source repository in "Package settings" at: `https://github.com/orgs/<org>/packages/nuget/<package>/settings`

Simply delete the old repository source and add the new repository under Manage Actions access.
